github3/pulls.py

```diff
@@ -286,6 +286,40 @@ def create_review_comment(self, body, commit_id, path, position):
         json = self._json(self._post(url, data=data), 201)
         return self._instance_or_null(ReviewComment, json)
 
+    @requires_auth
+    def create_review(self, body, commit_id=None, event=None,
+                      comments=None):
+        """Create a review comment on this pull request.
+
+        :param str body:
+            The comment text itself, required when using COMMENT or
+            REQUEST_CHANGES.
+        :param str commit_id:
+            The SHA of the commit to comment on
+        :param str event:
+            The review action you want to perform. Actions include
+            APPROVE, REQUEST_CHANGES or COMMENT. By leaving this blank
+            you set the action to PENDING and will need to submit the
+            review. Leaving blank will result in a 422 error response
+            which will need to be handled.
+        :param list comments:
+            Array of draft review comment objects.
```

Comment:
What does a draft review comment object look like? Is someone supposed to call `create_review_comment` a bunch and do something with that?

Reply:
Let's expand that the review comment objects are documented in https://developer.github.com/v3/pulls/reviews/#create-a-pull-request-review and as of the time of writing were dictionaries with 3 keys: path, position, body